protocol/src/main/protobuf/User.proto

```diff
@@ -72,6 +79,7 @@ message UserToBitHandshake {
   optional bool support_complex_types = 6 [default = false];
   optional bool support_timeout = 7 [default = false];
   optional RpcEndpointInfos client_infos = 8;
+  optional SaslSupport sasl_support = 9;
```

Comment:
Meant more like what level of "sasl support", since there could be more than two.